### PR TITLE
chore: convert sql glossary to markdown

### PR DESCRIPTION
**Pull request by schalkneethling**
This converts the SQL glossary page from HTML to Markdown for `es`

### CODE REVIEW

File: files/es/glossary/sql/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: SQL
+slug: Glossary/SQL
+tags:
+  - BBDD
+  - Glosario
+  - Sql
```
Comment (JuanVqz) [inline files/es/glossary/sql/index.md:7]: we are not adding tags anymore. 

```suggestion
```

File: files/es/glossary/sql/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+translation_of: Glossary/SQL
+---
```
Comment (JuanVqz) [inline files/es/glossary/sql/index.md:9]: We are adding the last sha reference on content repo https://github.com/mdn/content/blob/main/files/en-us/glossary/sql/index.md

```suggestion
l10n:
  sourceCommit: d2a9f2e26a8139d4bb270d7dc3cddd8b848719fe
---
```

File: files/es/glossary/sql/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+## Saber más
+
+### Conocimiento general
+
+- [SQL](https://es.wikipedia.org/wiki/SQL) en Wikipedia
+
+### Aprende SQL
+
+- [Aprende SQL en sqlzoo.net](http://sqlzoo.net/wiki/SQL_Tutorial)
+- [Tutorial Point](http://www.tutorialspoint.com/sql/)
```
Comment (JuanVqz) [inline files/es/glossary/sql/index.md:21]:
```suggestion

## Véase también
- [SQL](https://es.wikipedia.org/wiki/SQL) en Wikipedia
- [Aprende SQL en sqlzoo.net](http://sqlzoo.net/wiki/SQL_Tutorial)
- [Tutorial Point](http://www.tutorialspoint.com/sql/)

```

**Review by JuanVqz — COMMENTED** [review]
@schalkneethling  Thank you for contributing to the ES community, you are welcome.
I did some suggestions, if you agree feel free to add them if not please share your thoughts.
:+1:

File: files/es/glossary/sql/index.md (hunk 1 above)
Comment (schalkneethling) [reply files/es/glossary/sql/index.md:7]: Ah, this was automatically added by the markdown conversion tool. I reckon we need to raise an issue about this or we will have to manually delete them. Thank you for highlighting this.

File: files/es/glossary/sql/index.md (hunk 2 above)
Comment (schalkneethling) [reply files/es/glossary/sql/index.md:5]: Ah, interesting, is this documented somewhere?

**Comment by schalkneethling** [comment]
@JuanVqz, this is ready for your review again.

File: files/es/glossary/sql/index.md (hunk 1 above)
Comment (JuanVqz) [reply files/es/glossary/sql/index.md]: thanks for testing with es files :+1:

File: files/es/glossary/sql/index.md (hunk 2 above)
Comment (JuanVqz) [reply files/es/glossary/sql/index.md:7]: yes, the main branch is here https://github.com/mdn/mdn-community/discussions/129#discussioncomment-3191138

**Review by JuanVqz — COMMENTED** [review]
Thank you for helping us :+1:

**Pull request merged**